rpc/connection.go

```diff
@@ -348,6 +333,27 @@ type ConnectionOpts struct {
 	DialerTimeout time.Duration
 }
 
+func NewTLSConnectionWithLogrus(
+	srvRemote Remote,
+	rootCerts []byte,
+	errorUnwrapper ErrorUnwrapper,
+	handler ConnectionHandler,
+	logFactory LogFactory,
+	logrusLogger LogrusLogger,
+	opts ConnectionOpts,
+) *Connection {
+	transport := &ConnectionTransportTLS{
+		rootCerts:  rootCerts,
+		srvRemote:  srvRemote,
+		logFactory: logFactory,
+		wef:        opts.WrapErrorFunc,
+		log:        newConnectionLogLogrus(logrusLogger, "conn_tspt"),
```

Comment:
Do we need to pass through `dialerTimeout` here? The functions below are inconsistent about that, and I'm not sure whether it's a mistake or not.

Reply:
Good catch. I copied from `NewTLSConnection` and it was not there either. So I guess we missed it in the first place. Will add it to both places.